corehq/apps/events/models.py

```diff
@@ -164,3 +169,33 @@ def get_by_event_id(cls, event_id, domain, only_ids=False) -> set[Attendee]:
             return referenced_case_ids
 
         return cls.get_by_ids(referenced_case_ids, domain)
+
+    def save(self, user_id):
+        if not user_id:
+            raise InvalidAttendee('Attendee must have user_id specified')
```

Comment:
Attendees don't have to be mobile workers. See "[Assumptions](https://docs.google.com/document/d/1hU2SQvnfkfxu3RoxhRCKL6IuWl6zO7nuZFEbgF5grrE/edit#heading=h.mtfd39lckcwc)" and the user story "[An agricultural training event](https://docs.google.com/document/d/1hU2SQvnfkfxu3RoxhRCKL6IuWl6zO7nuZFEbgF5grrE/edit#heading=h.as5own42v8pq)".